src/commands/Connection.ts

```diff
@@ -135,5 +148,29 @@ export /**
   add(path?: string): Observable<any> {
     return this.helperAdd.add();
   }
+  /**
+   * check if the connection add command is disabled
+   */
+  addEnabled(): boolean {
+    if (!RxFs.exist(`${process.cwd()}/connections`)) {
+      return false;
+    }
+    if (!this.userRc.server.length) {
```

Comment:
- this.userRc.server.length check kann vor dem exists check.
Vielleicht so:

if (!this.userRc.server.length) {
  return false;
}
return RxFs.exist(`${process.cwd()}/connections`;